index.html

```diff
@@ -38,162 +44,182 @@ <h3 class="cover-links">LET'S CONNECT</h3>
-        <li><a href="#" class="fa fa-google"></a></li>
-        <li><a href="#" class="fa fa-github"></a></li>
-        <li><a href="#" class="fa fa-javascript"></a></li>
-      </ul>
-      <button class="resume">Get My Resume</button>
-      <h3 class="languages">Languages</h3>
-
-      <img src="images/skills/html.svg" alt="html" />
-      <img src="images/skills/css.svg" alt="css" />
-      <img src="images/skills/js.svg" alt="js" />
-
-      <div class="frameworks">
-        <div class="framework1">Framework</div>
-        <div class="greater-tag">></div>
+      <div class="about-myself-content">
+          <h1 class="about-myself-title">
+            About <br />
+            Myself
+          </h1>
+          <p class="myself-content">
+            Hello I am a software develper. I can help you build a product, feature
+            or website. Look through some of my work and expereince. If you like
+            what you see and have a project you need coder, don't husitate to
+            contact me.
+          </p>
+          <h3 class="links">LET'S CONNECT</h3>
+          <ul class="socials">
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#" class="fa fa-linkedin"></a></li>
+            <li><a href="#" class="fa fa-google"></a></li>
+            <li><a href="#" class="fa fa-github"></a></li>
+            <li><a href="#" class="fa fa-javascript"></a></li>
+          </ul>
+          <button class="resume">Get My Resume</button>
       </div>
-      <hr>
-      <div class="frameworks">
-        <div class="framework1">Skills</div>
-        <div class="greater-tag">></div>
+      <div class="about-myself-languages">
+          <h3 class="languages">Languages</h3>
+          <div class="framework-skills">
+            <img src="images/skills/html.svg" alt="html" />
+            <img src="images/skills/css.svg" alt="css" />
+            <img src="images/skills/js.svg" alt="js" />
+          </div>
+          <div class="framework-skills-desktop">
+            <img class="html-skill" src="images/Desktop_skills/Frame 9.svg" alt="html" />
+            <img class="css-skill" src="images/Desktop_skills/Frame 10.svg" alt="css" />
+            <img class="js-skill" src="images/Desktop_skills/Frame 11.svg" alt="js" />
+          </div>
+          <div class="frameworks">
+            <div class="framework1">Framework</div>
+            <div class="greater-tag">></div>
+          </div>
+          <hr>
+          <div class="frameworks">
+            <div class="framework1">Skills</div>
+            <div class="greater-tag">></div>
+          </div>
+          <hr>
       </div>
-      <hr>
+
+      <!-- Contact section -->
     </section>
     <footer class="contact">
-      <h1 class="contact-title">Contact me</h1>
+      <div class="footer-content">
+        <h1 class="contact-title">Contact me</h1>

```

Comment:
- Please give your form inputs a little bit more width for your desktop version 🙏 🙏. 

Your Project:
<img width="873" alt="dd" src="https://user-images.githubusercontent.com/73607512/199808800-18e9c60e-8cfb-4be9-82a9-675be8c13e1a.PNG">

Design Template:
<img width="250" alt="qwec" src="https://user-images.githubusercontent.com/73607512/199808971-c318e8fe-313b-45d8-b9b3-0e9ddf72f4dc.PNG">

Reply:
- I think your form will look much better if you increased the width a bit. I think you fix this by removing or reducing  the `padding: 2.5rem` added to` .my-form` on the desktop screen.

-  Having `padding: 2.5rem` on mobile is perfect. 🍷

Reply:
:+1: